freezegun/api.py

```diff
@@ -429,6 +434,15 @@ def stop(self):
         time.localtime = time.localtime.previous_localtime_function
         time.strftime = time.strftime.previous_strftime_function
 
+    def increment(self, delta=datetime.timedelta(seconds=1)):
+        if self.tick:
+            raise TypeError("Cannot manually increment when tick is True.")
```

Comment:
Not happy with having this here, IMO the different frozen time classes should implement `increment` or not on their own, but that would require more extensive refactoring.